public/css/global.css

```diff
@@ -1,4 +1,36 @@
 html, body {
     margin: 0;
     padding: 0;
-}
\ No newline at end of file
+}
```

Comment:
This file is for "global" CSS only, so if you have some CSS for specific elements, use variables inside the file of the components.